content/en/docs/Concepts/olm-architecture/dependency-resolution/_index.md

```diff
@@ -510,8 +535,8 @@ OLM performs dependency resolution at the namespace scope. It is possible to get
 
 ## Backwards-Compatibility Notes
 
-`dependencies.yaml` is supported in `0.16.1+` versions of OLM.
+`dependencies.yaml` is supported in `0.16.1+` versions of OLM and `1.10.0+` versions of OPM.
```

Comment:
nit: last 0.x -> 1.x substitution.

```suggestion
`dependencies.yaml` is supported in `1.16.1+` versions of OLM and `1.10.0+` versions of OPM.
```

I think this lgtm to me elsewise.

Reply:
@timflannagan I know this is confusing.

All operator-lifecycle-manager/OLM releases are 0.x releases.
All operator-registry/opm releases are 1.x releases.

I think what I have is accurate.

Dependencies.yaml support and the addition of package dependencies was added to both projects at different versions.

Reply:
Yeah, you're right. It looks like I glossed over the "versions of OLM" part and my brain mapped that to "versions of OPM"

/lgtm